client/signup/steps/website-content/section-types/contact-page-details.tsx

```diff
@@ -28,6 +31,10 @@ export function ContactPageDetails( {
 				value={ page.content }
 				error={ formErrors[ 'content' ] }
 				label={ description }
+				characterLimit={ CHARACTER_LIMIT }
+				characterLimitError={ translate(
+					'You’ve exceeded the character limit for this section. Please reduce the text to under 5000 characters for optimal presentation. If the text remains over this limit, we will optimize it with AI for you.'
+				) }
```

Comment:
Out of curiosity, is there any reason for you using the variable in the other form but not here? Might be beneficial if the number changes in future to prevent re-translation? (I totally might be missing something here though!!)

Reply:
Good catch, it was missed in the original commit!